api/v1alpha1/flowcollector_types.go

```diff
@@ -366,7 +366,19 @@ type FlowCollectorFLP struct {
 	Env map[string]string `json:"env,omitempty"`
 }
 
+const (
+	HPAStatusDisabled = "DISABLED"
+	HPAStatusEnabled  = "ENABLED"
+)
+
 type FlowCollectorHPA struct {
+	// +kubebuilder:validation:Enum:=DISABLED;ENABLED
+	// +kubebuilder:default:=DISABLED
+	// AuthToken describe the way to get a token to authenticate to Loki
```

Comment:
wrong description, I bet you copy-pasted :D

Reply:
Yes sorry, fixed.